content/docs/beyond-the-basics/public-attestation/index.md

```diff
@@ -1,14 +1,175 @@
+
+Any Events updating a Public Asset will also be public, and will each have their own unique Public URL.
+
+Following the link to a Public Asset or Public Event will allow read-only access to its information, without the need to sign in to RKVST.
+
+
+## Creating a Publicly Attested Asset
+
+{{< warning >}}
+**Warning**: Assets can only be made public at Asset Creation and cannot be made private afterwards.
+{{< /warning >}}
+
+1. Create an Asset with your desired attributes. See [Creating an Asset](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/) for detailed instructions. 
+
+{{< tabs name="create_asset_public" >}}
+{{{< tab name="UI" >}}
+Select `Add Asset` from the sidebar and fill in the desired details.
+{{< img src="CreateAsset.png" alt="Rectangle" caption="<em>Asset Details</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Create a JSON file with your desired Asset details. 
+
+```json
+{
+    "behaviours": ["RecordEvidence", "Attachments"],
+    "attributes": {
+        "arc_display_name": "Publicly Attested Asset",
+        "arc_display_type": "Example",
+        "arc_description": "This example asset is publicly attested, so anyone with the link can access its details without signing in to RKVST."
+    }
+}
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+2. Set your Asset to public.
+
+{{< warning >}}
+**WARNING:** Once an Asset is made public, it cannot be made private. 
+{{< /warning >}}
+
+{{< tabs name="set_public_public" >}}
+{{{< tab name="UI" >}}
+Check the box next to `Make Asset Public`, then click `Create Asset` to complete your Public Asset creation. 
+{{< img src="PublicCheck.png" alt="Rectangle" caption="<em>Check Asset as Public</em>" class="border-0" >}}
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Set keyword `public` to true. 
+
+```json
+{
+    "behaviours": ["RecordEvidence", "Attachments"],
+    "attributes": {
+        "arc_display_name": "Publicly Attested Asset",
+        "arc_display_type": "Example",
+        "arc_description": "This example asset is publicly attested, so anyone with the link can access its details without signing in to RKVST."
+    },
+    "public": true
+}
+```
+Use the curl command to run your JSON file. See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here. 
+
+```bash 
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v2/assets
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+3. Retrieve public link to share your Public Asset with others. 
+
+{{< note >}}
+**NOTE:** Public Assets may only be updated by the creating tenancy. Anyone viewing the Asset using the public link will have read-only access.
```

Comment:
Is "creating tenancy" used elsewhere in the docs? Is there a better wording for this? Asset owning tenancy?

Reply:
Changed to "A Public Asset may only be updated by the tenancy that created it. Anyone viewing the Asset using the public link will have read-only access" so it is more clear, as "creating tenancy" isn't used elsewhere.